### PR TITLE
Ava/intro

### PR DESCRIPTION
**Pull request by avahoffman**
Beefing up some stuff in the intro including material from the slides here: https://sharing.nih.gov/sites/default/files/flmngr/DMS-policy-slides.pptx

Please take a look at the TODO - I wasn't sure about these points.

### CODE REVIEW

File: 01-intro.Rmd [hunk 1]
```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
...
+The NIH Data Management and Sharing (DMS) Policy applies to **all NIH-supported research generating scientific data**. But what is "scientific data"?
+
+**SCIENTIFIC DATA**
+
+Scientific data are the “recorded factual material of sufficient quality to validate and replicate research findings, regardless of whether the data are used to support scholarly publications”. This can include any of the following if they are applicable to your study:
+
+- unpublished results
+- null results
+- results used to publish papers
+<!-- TODO: are the following two points correct? -->
```
Comment (avahoffman) [inline 01-intro.Rmd:79]: I think this is a question we're going to get, but I don't know the answer for raw or cleaned data.

File: 01-intro.Rmd [hunk 2]
```diff
@@ -16,15 +16,37 @@ First we will discuss the motivations behind the new data management and sharing
 
 So why is the NIH doing this? There are several reasons why sharing data can be beneficial to the scientific community.
 
-- **Supports transparency** - Sharing data provides more clarity about how studies are performed. 
--  **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
+- **Supports transparency** - Sharing data provides more clarity about how studies are performed. Many scientists also believe in an ethical responsibility to study participants (@Bauchner_citation_2016).
```
Comment (carriewright11) [inline 01-intro.Rmd:19]: Nice! Yeah, I think we could add more references...

File: 01-intro.Rmd [hunk 3]
```diff
@@ -16,15 +16,37 @@ First we will discuss the motivations behind the new data management and sharing
...
 - **Supports multi-modal work** - When more data of various types are easily available it makes it easier for scientist to perform studies with multiple types of data. 
 - **More efficient and cost effective** - Some data are especially difficult or expensive to produce.
 - **Supports Researcher Inclusion** - Data generation can be especially difficult for those at institutes with less resources. Publicly available data can therefore be used by these researchers to better enable their participation. 
 - **Increased impact** - Papers that share their data in repositories appear to be cited more based on the study by @colavizza_citation_2020
 - **Increased collaboration opportunity** - Having data available can encourage other researchers to expand the research in a new direction or extend it further and they may reach out to collaborate.
 - **Data Citations** - Due to the importance of data generation and sharing to the NIH, data will now be seen as research product that demonstrates a contribution to the scientific community.
 
+## Key terms
```
Comment (carriewright11) [inline 01-intro.Rmd:28]: Love this!

File: 01-intro.Rmd [hunk 4]
```diff
@@ -16,15 +16,37 @@ First we will discuss the motivations behind the new data management and sharing
...
 - **Increased collaboration opportunity** - Having data available can encourage other researchers to expand the research in a new direction or extend it further and they may reach out to collaborate.
 - **Data Citations** - Due to the importance of data generation and sharing to the NIH, data will now be seen as research product that demonstrates a contribution to the scientific community.
 
+## Key terms 
+
+<div class = "dictionary">
+
+**Data Management**
+
+The process of validating, organizing, protecting, maintaining, and processing scientific data to ensure the accessibility, reliability, and quality of the scientific data for its users. All data should be managed.
```
Comment (carriewright11) [inline 01-intro.Rmd:34]: have the word process and processing...(maybe want to use something else?) tricky though

File: 01-intro.Rmd [hunk 5]
```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
 
 For certain grant mechanisms for projects that do not generate data, compliance with the policy is not required. For certain types of data, sharing is not possible, and a justification will be required instead. 
 
-What about grant renewals? 
+The following outline several key questions:
+
+- **Does my research generate scientific data?** If yes, you will need to submit a DMS Plan.
```
Comment (carriewright11) [inline 01-intro.Rmd:64]: I'm still unclear if this is true for some mechanisms

File: 01-intro.Rmd (hunk 1 above)
Comment (carriewright11) [inline 01-intro.Rmd:79]: hmm I would have to check the presentations again...

File: 01-intro.Rmd [hunk 6]
```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
...
+## Can I elect to not share data?
+
+There are legitimate reasons you might not want to share your data. Data might not be sharable due to ethical, legal, or technical concerns. **You will still need to submit a DMS Plan even if you plan to withold data sharing**. You must explain your reasoning in your DMS Plan.
+
+Justifiable ethical, legal, and technical factors for limiting sharing of data include: 
 
-What about no cost extensions?
+- Informed consent will not permit or will limit the scope or extent of sharing and future research use
+- Existing consent (e.g., for previously collected biospecimens) prohibits sharing or limits the scope or extent of sharing and future research use
+- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient
```
Comment (carriewright11) [inline 01-intro.Rmd:124]: I think we should put a disclaimer about this- that we suggest they hire an expert to help de-identify the data - maybe we can point them to ideas of where to find such people too?

File: index.Rmd [hunk 7]
```diff
@@ -21,24 +21,25 @@ knitr::write_bib(c(
 
 # About this Course {-}
 
-The [National Institutes of Health (NIH)](https://www.nih.gov/) is rolling out a new policy for the management and sharing of scientific data starting for (most) grants submitted January 25, 2023 or later.
+The [National Institutes of Health (NIH)](https://www.nih.gov/) is rolling out a new policy for the management and sharing of scientific data starting for (most) grants submitted **January 25, 2023** or later.
```
Comment (carriewright11) [inline index.Rmd:24]: Nice - definitely good to emphasize it!

**Review by carriewright11 — APPROVED** [review]
Looks great! I will see what I can do to help with the questions you had

File: 01-intro.Rmd (hunk 1 above)
Comment (carriewright11) [reply 01-intro.Rmd:79]: right... it's an important question

File: 01-intro.Rmd (hunk 4 above)
Comment (avahoffman) [reply 01-intro.Rmd:34]: lifted straight from NIH - LOL but I will change it :)

File: 01-intro.Rmd (hunk 1 above)
Comment (carriewright11) [reply 01-intro.Rmd:79]: watching it again... I think it is whatever you are able to share - so if you can share raw I think they prefer that - but maybe not required??? I think we might need to ask

**Review by carriewright11 — APPROVED** [review]

File: 01-intro.Rmd (hunk 2 above)
Comment (avahoffman) [reply 01-intro.Rmd:19]: What do you think about a page of citations that they can refer to in the plan?

File: 01-intro.Rmd (hunk 5 above)
Comment (avahoffman) [reply 01-intro.Rmd:64]: going off of their slide here. What do you think?
image

File: 01-intro.Rmd (hunk 2 above)
Comment (carriewright11) [reply 01-intro.Rmd:19]: oooh cool idea

File: 01-intro.Rmd (hunk 5 above)
Comment (avahoffman) [reply 01-intro.Rmd:64]: maybe "follow the policy?"

Comment (carriewright11) [reply 01-intro.Rmd:64]: yeah they said they would provide info about activity codes - hopeful that link will or already exists and we can just point them to that

File: 01-intro.Rmd (hunk 6 above)
Comment (carriewright11) [reply 01-intro.Rmd:124]: definitely!

File: 01-intro.Rmd [hunk 8]
```diff
@@ -124,7 +124,7 @@ Justifiable ethical, legal, and technical factors for limiting sharing of data i
 
 - Informed consent will not permit or will limit the scope or extent of sharing and future research use
 - Existing consent (e.g., for previously collected biospecimens) prohibits sharing or limits the scope or extent of sharing and future research use
-- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient
+- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient. _At Fred Hutch, you should get in contact with the [Institutional Review Office](https://centernet.fredhutch.org/cn/u/iro.html){target="_blank"} if you have questions about compliance of your research._
```
Comment (avahoffman) [inline 01-intro.Rmd:127]: @carriewright11 how's this? @vortexing could use your Hutch wisdom here :)

Comment (carriewright11) [reply 01-intro.Rmd:127]: good - I just also want to scare people away from doing de-identification without expert help :P

Comment (avahoffman) [reply 01-intro.Rmd:127]: agreed! Probably best to get _something_ down now, then run it by Hutch experienced folks for a gut check.

**Pull request merged**